static/src/js/components/PermissionForm/PermissionForm.jsx

```diff
@@ -359,9 +359,15 @@ const PermissionForm = ({ selectedCollectionsPageSize }) => {
       const searchAndOrderGroupPermission = []
       const searchPermission = []
 
+      // Removes invalid group permission items, which has no id and no userType
+      // eslint-disable-next-line max-len
+      const removeInvalidGroupPermissionItems = (groupPermissionItems) => groupPermissionItems?.filter((item) => item.id || item.userType)
```

Comment:
I think you should add a test for this logic as well.

Reply:
Added